File: ProjetASI2/Personal-Finance-Management-Spring-boot/src/main/java/com/PersonalFinanceManagementApplication/Personal/Finance/Management/Application/dataTransferObject/LoginResponseDto.java
package com.PersonalFinanceManagementApplication.Personal.Finance.Management.Application.dataTransferObject;

import com.PersonalFinanceManagementApplication.Personal.Finance.Management.Application.Model.User;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class LoginResponseDto {
    private String token;
    private User user;
    private String message;
}

File: ProjetASI2/PersonalFinanceManagementAppFrontEnd/src/app/Model/LoginResponseDto.ts
import { User } from "./User";

export class LoginResponseDto{
    token:String;
    message:String;
    user: User| null;

    constructor(token:String, message:String, user:User| null){
        this.token = token;
        this.message = message;
        this.user = user;
    }
}
